blazor-toc.html

```diff
@@ -2391,6 +2391,34 @@
 			</li>
 		</ul>
 	</li>
+	<li>Pager
+	    <ul>
+			<li>
+			    <a href="/blazor/pager/getting-started">Getting Started</a>
```

Comment:
Navigation -> methods are not covered.
https://github.com/essential-studio/ej2-blazor-source/blob/development/Syncfusion.Blazor/Navigations/Pager/SfPager.razor.method.cs